internal/audit-log/command_config_migrate.go

```diff
@@ -79,8 +79,8 @@ func (c *configCommand) migrate(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 	for _, warning := range warnings {
-		output.ErrPrintln(warning)
-		output.ErrPrintln()
+		color.ErrPrintln(c.Config.EnableColor, warning)
+		color.ErrPrintln(c.Config.EnableColor, "")
```

Comment:
It looks like the `output` package is still around in this PR and being used in a few places; what's the general guideline for when to use `output` and when to use `color`? Or could we fully replace output with color?

Reply:
+1

Reply:
Good idea, updated to fully replace color with output to avoid future confusion!